bluemira/power_cycle/net/manager.py

```diff
@@ -370,83 +302,43 @@ class PowerCycleManager:
         each Power Cycle load group.
     """
 
-    # ------------------------------------------------------------------
-    # CLASS ATTRIBUTES & CONSTRUCTOR
-    # ------------------------------------------------------------------
-
-    _load_types = PowerCycleSystem.list_all_load_types()
-
-    _manager_format = FormattedLibrary.Format(
-        {
-            "name": str,
-            "config_path": str,
-            "systems": list,
-        }
-    )
-
     def __init__(self, scenario_config_path: str, manager_config_path: str):
-        scenario_builder = ScenarioBuilder(scenario_config_path)
-        self.scenario = scenario_builder.scenario
-
-        validated_path = validate_file(manager_config_path)
-        json_contents = read_json(validated_path)
-
-        self._manager_config = FormattedLibrary(
-            self._manager_format,
-            dictionary=json_contents,
-        )
-
-        self.group_library = self._build_group_library(
-            self.scenario,
-            json_contents,
-        )
-
-    @staticmethod
-    def _build_group_library(scenario, manager_config):
-        group_library = Library(PowerCycleGroup)
-        for group_label, group_inputs in manager_config.items():
-            group_name = group_inputs["name"]
-            group_config_path = group_inputs["config_path"]
-            group_systems = group_inputs["systems"]
+        self.scenario = ScenarioBuilder(scenario_config_path).scenario
+        self.manager_configs = {
+            key: PowerCycleManagerConfig(**val)
+            for key, val in read_json(validate_file(manager_config_path)).items()
+        }
 
-            validated_path = validate_file(group_config_path)
-            json_contents = read_json(validated_path)
+        self.group_library = self._build_group_library()
 
-            group_config = {key: json_contents[key] for key in group_systems}
+    def _build_group_library(self):
+        group_library = {}
+        for group_label, group_inputs in self.manager_configs.items():
+            group_config = {
+                key: read_json(validate_file(group_inputs.config_path))[key]
+                for key in group_inputs.systems
+            }
 
             group = PowerCycleGroup(
-                group_name,
-                scenario,
+                group_inputs.name,
+                self.scenario,
                 group_config,
                 label=group_label,
             )
             group_library[group_label] = group
         return group_library
 
-    # ------------------------------------------------------------------
-    # OPERATIONS
-    # ------------------------------------------------------------------
-
     def _build_pulseload_of_type(self, load_type):
```

Comment:
To match my previous comment, `load_type` should be changed to `load_classification`. However, as I mentioned in the general comment, I think that this information should be stored as an attribute of PhaseLoad (PulseLoad? ScenarioLoad? To be discussed!) objects, established during creation with the inputs found in `VariablesMapConfig`. This would mean that the `_build_pulseload_of_type/classification` would find all information necessary on the loads themselves.